example/src/HelloWorldOCaml.ml

```diff
@@ -1,3 +1,51 @@
-[@@@react.dom]
+open React.Dom.Dsl
+open Html
 
-let%component make () = div ~children:[React.string "Hello world from OCaml"] ()
+let%component make () =
+  fragment
+    [
+      a
+        [|
+          href "https://github.com/jihchi/jsoo-react-realworld-example-app";
+          target "_blank";
+        |]
+        [ i [|className "ion-social-github"|] []; string "Fork on GitHub" ];
+      footer [||]
+        [
+          div
+            [|className "container"|]
+            [
+              span
+                [|className "attribution"|]
+                [
+                  string "An interactive learning project from ";
+                  a [|href "https://thinkster.io"|] [ string "Thinkster" ];
+                  string ". Code &amp; design licensed under MIT.";
+                ];
+            ];
+        ];
+    ]
+
+let web_component =
```

Comment:
Or should I put it in a separate example, to keep this simple?

Reply:
Do we want to pull in leaflet and the leaflet web components just to demo this though? Or should we demo it some other way?

Reply:
> Or should I put it in a separate example, to keep this simple?

Yes, this sounds great.

> Do we want to pull in leaflet and the leaflet web components just to demo this though? Or should we demo it some other way?

If you think it's ok yes :) I think demoing leaflet would be really cool. It could all go into a "JS bindings" category (but I also don't want to hold this PR to lead to more conflicts / time spent for you 🤗 ). Up to you completely

Reply:
If you think it's OK to merge as=is, I'd be happy to follow up on this later.